src/neuroconv/datainterfaces/ecephys/spike2/spike2datainterface.py

```diff
@@ -10,7 +10,7 @@ def _test_sonpy_installation() -> None:
     get_package(
```

Comment:
For my own curiosity, how did we ended up with this test here for `spike2`? This is exclusive for this datainterface?

Reply:
You mean the architecture subkey, or the `get_package` helper in general? Both have been around a while so they must have been for something else. Useful in general for any external import that has trouble with certain Python versions or platforms

Reply:
I realized I quote the wrong line. I am talking about the `_test_sonpy_installation()` function as a whole.